### DISCUSSION TITLE
Who wants to be hired? (Jan 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#32938] [Ideas]
# What is this?
Happy New Year everyone! This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #32094. We've had more [*success stories*](https://twitter.com/Nutlope/status/1475929984174895105) of developers being hired!*

If you're a company looking to post a role, use #32937.

**Comment by azakero** [comment]
Hello.

Location: Bangladesh
Remote: Yes
Willing to relocate: Yes
Technologies: NodeJS, ExpressJS, Koa, MongoDB, Mongoose, Firebase, CSS3, Javascript, jQuery, ReactJS, Redux, NextJS, Material UI, Tailwind CSS
Resume/CV: https://drive.google.com/file/d/1oKX6L4-or6hAbVxP01qf68uio9-wRnzD/view
Website: https://www.zaker.dev
Email: gloryfaircompany@gmail.com

Thank you.

**Comment by markpsyntax** [comment]
Location: Minneapolis, MN
Remote: Yes
Willing to relocate: Yes
Technologies: Next, React, Angular, JavaScript, Node, Express, Tailwind, GraphQL, AWS
Resume/CV: https://drive.google.com/file/d/18iPywvUOhRJbgYWu2jhyHZNAzS905-IB/view?usp=sharing
Website: https://markphillips.vercel.app
Email: mphill05@gmail.com

**Comment by developerisgood** [comment]
Location:  LOS ANGELES	CA
Remote: Yes
Willing to relocate: Yes
Technologies:  JavaScript, React, NextJS, GatsBy, AWS, Node, Express, MySQL,  TypeScript, Srapi, Algorand, Blockchain
Resume/CV: https://drive.google.com/file/d/1g6FKmjBLqccOk6710LNjDYEbVuC3d6gu/view?usp=sharing
Website: https://www.linkedin.com/in/codepro/
Email: farlight216@gmail.com

**Comment by talentjo** [comment]
Location: Mexico City, MX
Remote: Yes
Willing to relocate: Yes
Technologies: HTML/CSS. JavaScript, React, NextJS, Vue, TypeScript, Node, Express, AWS, GraphQL, MySQL, MongoDB, smartcontract, Defi, Solana 
Resume/CV: https://drive.google.com/file/d/1sdUb4xAeHCxnAzs4N0gLqqWA-InjoA7u/view?usp=sharing  
Website: https://michaeldev.herokuapp.com/
Email: joseph.code.pro@gmail.com

**Comment by simonhlee97** [comment]
Location: Incheon, Korea
Remote: Yes
Willing to relocate: Yes
Technologies: HTML/CSS, JavaScript, React, NextJS, Vue, TypeScript, Node, Express, Postgres, MongoDB
Resume: https://www.dropbox.com/s/9o71zjc30tu9bd3/resume.simon.lee.pdf?dl=0
Website: https://simonhlee97.github.io/
Email: simonhlee4945@gmail.com

**Comment by fullDev9233** [comment]
### Looking for a remote Frontend/Blockchain opportunity



**Location**: Ukraine

**Remote**: Remote Only

**Willing to relocate:** No

**Technologies**: React, Redux, Redux-Saga, Styled-Components, Storybook, Material UI, Typescript, ES6+, Web3, Ether.js, IPFS(Moralis), Vue, Vuex, Vuetify, BootstrapVue, Node

**Resume/CV**: https://drive.google.com/file/d/1xJBgQZAqh3yqepMgK4y9la7loIvUumDH/view?usp=sharing

**Website**: https://angel.co/u/maznichenko-anatolii

**Email**: mazanatolii9300@gmail.com

**Discord**: Developer9300#3096

**Comment by chemiadel** [comment]
Location: Algeria
Remote: Yes
Willing to relocate: Yes
Technologies: JS/TS, ReactJS, NodeJS, NextJS, MUI, Tailwindcss, Graphql (Apollo server/client), ExpressJS, Firebase, Deno
Others: Google Script, VBA, Wix velo
Résumé/CV: https://my.indeed.com/p/adelc-s89zs7b
Website: https://adel-portfolio.vercel.app/
Email: pace3man@gmail.com || adelchemi6@gmail.com

**Comment by Christopher-Stevers** [comment]
Location: Kitchener, Ontario
Remote: Yes
Willing to relocate: No.
Technologies: HTML, CSS, JavaScript, React.js, Next.js, Redux, Sass, Node.js, Express, MongoDB, Webpack, PostgreSQL, Git.
Website: https://devstevers.com
Resume: https://docs.google.com/document/d/1Oq5E2CZu8XSuWbmL8iF1CWhD5UHIQLdCPOiEiQvxu5Q/edit?usp=sharing
Email: christopherstevers@protonmail.com

**Comment by lamanIbrahimli** [comment]
Hello, Hope you are doing great. Looking for a remote Frontend opportunity
Location: Baku Azerbaijan
Remote: Yes
Willing to relocate: U.S. only (Willing to provide relocation assistance)
Technologies: HTML, CSS, JavaScript, React.js, Next.js, Redux, Sass,  Git, Java, Spring boot, Vue, Vuex, Vuetify, BootstrapVue,
Website: https://www.linkedin.com/in/leman-ibrahimli-b535091b3/
Resume: https://docs.google.com/document/d/1oU4sxGyO07NbYHBDxPuLxwi_66tvJ7c1grBdhwU5YMI/edit?usp=sharing
Email: leman.ibrahimli23@gmail.com

**Comment by GreatWorldDev** [comment]
Frontend Blockchain Developer role
Location: Japan
Remote: Yes
Technologies: React.js, Next.js, Redux, Sass, Git, Node, Nest, TypeOrm, Postgres
Website: 
https://suzk-dev.web.app/
https://www.linkedin.com/in/hayato-suzuki-2418a11a3/        
Email: great.world.dev@gmail.com

**Comment by booleanrecep** [comment]
Location: Istanbul / Turkey
Remote: Yes
Willing to relocate: Yes
Technologies:React,Redux,Next,Vue,Nuxt. Looking also for work or internship in Golang (beginner),Node(beginner).
Resume/CV: https://docs.google.com/document/d/1bCS7AmJTvTgb57JCKVafu1zxq0wFMInNsKAZZpEb9LY/edit?usp=drivesdk
Website:
Email: recep.fed@gmail.com

**Comment by KyleMcKell** [comment]
Location: Fort Collins, CO USA
Remote: Yes!
Willing to relocate: Depends on location
Technologies: CSS, TailwindCSS, React, Next, TypeScript
Website: https://www.kylemckell.com/
Email: me@kylemckell.com
[Resume](https://github.com/vercel/next.js/files/7834297/KyleMcKellResume2022.pdf)

**Comment by Hamza-DA** [comment]
Location: Morocco/Kenitra
Remote: Yes
Willing to relocate: Yes
Technologies: ReactJs, NextJs, Javascript, HTML, CSS, TailwindCSS, Bootstrap, FramerMotion.
Resume/CV: hamzadahmani.vercel.app/resume.pdf
Website: https://hamzadahmani.vercel.app
Email: madebyhamzadahmani@gmail.com

**Comment by city17** [comment]
### Hi, I'm Robbert and I'm looking for a remote front-end opportunity. 

### I build Next.js sites and apps, with a focus on interactive UIs and smooth animations.

Location: The Netherlands
Remote: Yes
Willing to relocate: No
Technologies: Next.js, Animations (Framer Motion / React Spring), GatsbyJS, React, Material UI, Contentful & Wordpress as headless backends, CSS.
Resume: robbertvanderhoeven.nl
Website: https://studiokoro.com
Email: robbert@studiokoro.com

**Comment by ghost** [comment]
Hi,

I am a senior full-stack engineer with years of experience building **web applications from scratch**.
I specialize in building high-performance, Search engine optimized **PWAs**.

_**Why Hire me?**_

- I am an expert in creating and implementing offline web apps that can be **installed** on mobile devices as well as on computers.
- **Performance** optimize your web apps to lightning-fast speed.
- Build complete web apps **Independently** without assistance.
- Revamp your website to Enhance the **SEO** of your site for better ranking in Search engines.
- Adept at building high **accessibility** web apps.
- Experienced in **Internationalization** and various **payment** **gateway** integrations like PayPal and Stripe.

_Backend dev experience:_ Setting up **Production and Development Environment**, Payment gateway Integration, **Database Architecture**, Backend Cyber security, authentication, etc.

_Front End experience:_ Creating Installable Web apps, SEO, **WCAG, and ADA compliant apps**, **Offline Web apps**, Performance optimization.


_Location_: India

_Remote_: Yes

_Willing to relocate_: Yes

_Technologies:_  **NEXT.js, React**, Redux, **TypeScript**, Firebase, Bootstrap, Material-UI

_Website:_ [Site](https://www.appusrn.in)

_Email_:  `appuauth@gmail.com`

_Discord_: `Auth#4881`

**Comment by crackwebai1024** [comment]
Remote : Yes
Willing to relocate: No
working timezone : client's timezone (including UK, EST, CST, PST)
Looking for : fully remote job (full-time)
mainskills : 
 - Frontend 
     (SPA - React, Angular, Vue, Svelte)
     (SSR - Next, Nuxt, Sapper)
     (UI - material ui, tailwind CSS, Chakra UI, styled-components, vuetify)
 - Backend (Node - express, Python - django, aws Lambda(chalice, nodejs))
portfolio : http://mingxing.vercel.app
contact: piaobristar@gmail.com

**Comment by zhyd1997** [comment]
Location: China
Remote: Yes
Willing to relocate: No
Technologies: see more details at here: https://github.com/007tom
Resume/CV:TODO (Shortly, I just have 5 months full time job experience _**Remote**, and Frontend developer role_.)
Website: https://softmaple.xyz
Email: zhyd007@gmail.com

**Comment by Ckbhatia** [comment]
Location: Balotra, India
Remote: Yes
Willing to relocate: Yes
Technologies: React, Redux, NextJS, TypeScript, JavaScript, Node, ExpressJs, HapiJs, HTML, CSS, Styled components, Tailwind, Gatsby,  AWS, GraphQL, REST, MongoDB, PostgreSQL, Mongoose, TypeORM, Jest, Mocha, Story-book etc.
Resume/CV: https://docs.google.com/document/d/1Zd2Aei1jPJOx5gJIvbPyxyzg0qF3JU76AvwNWDNgi50/edit?usp=sharing
Website: https://ckportfolio.netlify.com
Email: chetansain86@gmail.com

**Comment by max-bot1** [comment]
Location: Dallas, TX
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, React, NextJS, Node, Express, MySQL, ArcGIS Suite, Heroku 
Resume/CV: https://docs.google.com/document/d/18LPULNOfPYuaXdA6_XjRlhR2Hjmeb-qu6t4IPiXan0M/edit
LinkedIn: https://www.linkedin.com/in/max-botwin/
Website: https://maxbotwin.netlify.app/
Email: maxfbotwin@gmail.com

**Comment by kellenwiltshire** [comment]
Location: Ontario Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Squareup/Shopify
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by Sobefy** [comment]
Hi Kellen,

Thanks a lot for getting in touch regarding this opportunity. I would love
to setup a quick call to review the project scope.

Is Monday at 12:00pm EST a good time for you?

Let me know and I’ll send over a calendar invite.

Kind regards,

On Sat, Jan 15, 2022 at 8:55 AM Kellen Wiltshire ***@***.***>
wrote:

> Location: Ontario Canada
> Remote: Yes
> Willing to relocate: No
> Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL,
> Squareup/Shopify
> Résumé/CV: Linkedin <https://www.linkedin.com/in/kellenwiltshire/>
> Website: kellenwiltshire.com
> Email: ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/vercel/next.js/discussions/32938#discussioncomment-1974696>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOEMOZGW2TCZGOHBVGHAXQDUWF4GLANCNFSM5LC6YQEA>
> .
> You are receiving this because you commented.Message ID: <vercel/next.
> ***@***.***>
>
-- 
[image: facebook] <https://facebook.com/sobefy>
[image: linkedin] <https://www.linkedin.com/company/sobefy/>
[image: instagram] <https://instagram.com/sobefyagency>
Oscar Diaz

Managing Director

Sobefy eCommerce
(305) 204-0588
***@***.***
sobefy.com
Create Your Own Free Signature
<https://www.hubspot.com/email-signature-generator?utm_source=create-signature>

**Comment by thevinter** [comment]
**Location:** Italy
**Remote**: Yes
**Willing** to relocate: Yes
**Technologies**: 
  * JS / React / Nextjs / Angular / dotnet / Kubernetes / DevOps / MongoDB
After being hired at my current job, in a couple of months, I got assigned as Team Lead on a project for Virgin Active. I set up from scratch a microservices architecture, a CI/CD pipeline and a GitOps environment with Jenkins X. Wrote many of the backend services in dotnet and a huge chunk of the Frontend codebase (initially in plain React and then migrated to NextJS). Now I'm still an architectural consultant for the project + I work on other websites and mobile apps.

Why hire me: I put great emphasis on **user experience, speed** and **good engineering**. I know what it means to write a **lasting codebase** and I know the importance of testing and refactoring. I'm a quick learner, love challenges and I'm able to work independently and fast.

Resume/CV: https://blog.thevinter.com/cv
Email: contact@thevinter.com

**Comment by HeyClos** [comment]
Location: Las Vegas, NV
Remote: Yes
Willing to relocate: Yes
Technologies: React, Nextjs, Nodejs, JS, HTML/CSS, python
Resume/CV: https://twitter.com/HeyClos/status/1469445536713359367/photo/1
Email: thiscarlos@gmail.com

<3

**Comment by abeciana1** [comment]
**Location:** currently based in Burlington, VT for now
**Remote:** Yes
**Willing to relocate:** Yes
Technologies: JavaScript, React, Gatsby, TypeScript, Next.js, Redux, GraphQL, Ruby, Ruby on Rails, PostgreSQL, HTML, CSS, Git, Netlify, Heroku, Vercel, and some Node.js/Express
**Resume/CV:** [Alex Beciana — Full Stack Software Engineer](https://drive.google.com/file/d/1fYN-OTgwVzbyUaraK6u_Xxp0q53OpvzC/view)
**Website:** https://alexbeciana.com
**Email:** alexander.beciana@gmail.com

**Comment by PratikAwaik** [comment]
Ideally, looking for Frontend engineer roles. Open to full stack engineering roles.

Location: India
Remote: Yes (preferred)
Willing to relocate: No

Technologies: React, Redux, Next (learning), Vue, Node, Express, MongoDB, Mongoose, PostgreSQL, Sequelize, GraphQL(Apollo client and Server), Git

Resume: [Link](https://drive.google.com/file/d/11CNyg64cRzAnr9I7yci0zZkMOgn3Clyw/view?usp=drivesdk) 

Email: pratikawaik125@gmail.com

**Comment by igbominadeveloper** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: Vue JS, React, Typescript, Nodejs, GraphQL, NextJS, TailwindCSS
Resume/CV: https://drive.google.com/file/d/1pqp5h1U0B7hkN2ResDGWhsn-3btZNiNV/view?usp=sharing
Website: https://favourafolayan.me
Email: favourafolayan@gmail.com

**Comment by intrvertmichael** [comment]
I am a passionate Front End Web Developer with experience in Education and Graphic Design looking to start my career to help bridge the gap between users and tools that can help them.

Location: New York
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, Typescript, React, Next Js, Ruby on Rails
Resume/CV: https://intrvertmichael.github.io/resume.pdf
Website: https://intrvertmichael.github.io
Email: michaelpaguay@gmail.com

**Comment by Rafi-99** [comment]
Hello everyone! 

My name is Rafi. I am an intrinsically motivated software engineer who loves building things for the web. I am looking to experience career growth in my next role and would love to put my Next.js skills to work!

Location: Minnesota, United States
Remote: Yes
Willing to relocate: Yes
Technologies: Java, JavaScript, React.js, Next.js, Node.js, Express.js, AWS, MongoDB
Resume/CV: [Md_Rafi_Résumé.pdf](https://github.com/vercel/next.js/files/7919866/Md_Rafi_Resume.pdf)
Website: [rafi-codes.dev](https://rafi-codes.dev)
Email: contact@rafi-codes.dev
LinkedIn: https://www.linkedin.com/in/rafi2018

Thank you for your time!

**Comment by anthonysidesap** [comment]
Anthonysidesapps

**Comment by swayam-coder** [comment]
Location: India
Remote: Yes
Willing to relocate: Yes
Technologies: Next JS, React, Redux, React Query NodeJS (Express JS), TypeScript/JavaScript, GraphQL, MySQL, MongoDB
Resume/CV: https://drive.google.com/file/d/1yp-36zKL_UxM_NIFgZ6d-XiaZEzDM_cp/view?usp=drivesdk
Email: swayam14feb@gmail.com

**Comment by fl3x1324** [comment]
### Atanas Minkov - Full-Stack

Location: Bulgaria GMT+2
Remote: Exclusively
Willing to relocate: No
Technologies: Git, Java, JavaScript, TypeScript, Relational Databases, Firebase, React, Next.js, Linux, CI/CD, Spring Boot
Resume/CV: [Link](https://www.icloud.com/iclouddrive/059CVlABVw0Ly9rWDSs7KJCcQ#Atanas_Minkov_-_Full-stack_Developer)
Website: [Recent side project](https://selfmade-fl3x1324.vercel.app/)
Email: atanas.minkov@hey.com

**Comment by shreyvijayvargiya** [comment]
shrey Vijayvargiya, Full-stack 
I have ** 4 years** of working experience in software development. 
I’ve developed Mobile applications, Web Applications, worked on ** Web3.0, Moralis **, ** APIs **, ** Databases ** and scale the applications to millions of users. 

I’ve good hands-on experience with **Figma** and **UI/UX** that helps me to manage and deliver it to the users with ease. I’ve experienced in developing products from scratch, managing and leading team as well as working remotely.

** Projects  ** - Some of my live projects are 
** CryptStream ** - All crypto stream at once place (https://crypt-stream.vercel.app/dashboard)
** iHateReading ** - Learn how products are developed professionally (https://www.ihatereading.in)
** Witness Invoice **- Manage, Create and Edit all your invoices in one place. (https://www.witnessinvoice.com)
** Search Currency Open API  **  - Search the currency by country name (https://ihatereading.in/projects/search-currency?type=api)

** Tech Stack ** - React, React Native, Firebase, Web3.0, Moralis, Github, Database, Node.js, Express, GraphQL

** Portfolio  ** - https://iamshreyvijayvargiya.notion.site/](https//iamshreyvijayvargiya.notion.site/)
** Github **- https://github.com/shreyvijayvargiya](https://github.com/shreyvijayvargiya)
** LinkedIn ** - https://www.linkedin.com/in/shrey-vijayvargiya-b62a3a105/](https://www.linkedin.com/in/shrey-vijayvargiya-b62a3a105/)
** Medium ** - https://medium.com/@shreyvijayvargiya26](https://medium.com/@shreyvijayvargiya26)

**Comment by AWIXOR-zz** [comment]
Location: Morocco
Remote: Yes
Willing to relocate: Yes
Technologies: Next JS, React, Redux, Recoil, TypeScript/JavaScript, Tailwind ...
Resume/CV: https://awixor.com/resume.pdf
Website: https://awixor.com/
Email: me.awixor@gmail.com

**Comment by ariesclark** [comment]
I'm Aries, a 19 year old Canadian citizen with a burning passion for software development and improving the state of the web. I started software development when I was quite young — 13 years old in fact. In those 6 years I've done so many different things ranging from developing Minecraft mods in Java, writing quite a few horrible PHP websites, and all the way to where I am now. As of late, my focus has been on using modern languages like TypeScript to create web based applications and services with React and Next.js among other libraries.

I'm always open for an opportunity. Whether you have a question or just want to say hi, 
I'll try my best to get back to you! Take the chance and message me.

---

Location: Alberta, Canada
Remote: Yes
Willing to relocate: If given the right opportunity, yes.
Technologies: 
1. TypeScript / JavaScript (ES6+)
4. React and related libraries
5. Next.js (obviously ❤️)
6. Tailwind CSS 
7. and much much more.

Website: https://ariesclark.com/
Email: ``me@ariesclark.com``

**Comment by Sobefy** [comment]
Are you ready to ride the eCommerce Wave in Miami? 🌊As a passionate eCommerce agency who loves organizing events, we know you&#x27;ll be thrilled to be part of eCommerce Wave Miami, the most electrifying eCommerce event of the year!📆 Date: April 12, 2024🏖️ Location: Climate and Innovation Hub in MiamiHere&#x27;s what you can expect at eCommerce Wave Miami:🌟 Industry Experts: Learn from the best in eCommerce and stay ahead of the game.🤝 Networking Opportunities: Connect with successful store owners and industry peers.🛠️ Workshops: Dive deep into the latest trends and strategies.🛍️ Successful Brands: Discover the secrets of thriving eCommerce businesses.🏢 Leading Companies: Explore partnerships and collaborations.🎵 Live Music: Groove to the beats of talented musicians.🍔 Great Food: Savor delicious dishes and satisfy your taste buds.🍹 Refreshing Drinks: Enjoy a variety of beverages as you unwind.This event is tailor-made for professionals like you who are passionate about modern websites, Shopify stores, and tech growth. It&#x27;s a golden opportunity to expand your knowledge, network with fellow enthusiasts, and have a fantastic time in the vibrant city of Miami Beach.Don&#x27;t miss out on this incredible event! Secure your spot now and be part of the eCommerce Wave Miami. Early bird tickets are available for a limited time.🎟️ Get Your&nbsp;&nbsp;Early Bird Tickets Below!Join us as we make waves in the eCommerce world! We can&#x27;t wait to see you there.Let&#x27;s ride the eCommerce Wave together!Warm regards,Sobefy eCommerceeCommerce Wave Miami Team



Featured Events:

eCommerce Wave Miami | Future of Cities Climate and Innovation HUB, Northeast 59th Street, Miami, FL, USA | 2024-04-12T13:00:00Z
https://www.eventbrite.com






Unsubscribe: https://www.eventbrite.com/organizations/missive/activity/unsubscribe/?p=ABIdvVvQsZmQDHa3UY5zXlOUpvYaZ7lBwRDVwvyLlsMMvOwtQ32bgPXchTbjr9HfouD4TaXgXO5TTimJ7wOd_Tz3Ld-3QpMKz6u6H_aDDhLvKuaFNV5YHAN4W90TReCEd-yvLRh86mmZ-Glc_HQ8BgB2oHv1xqneIbE5ZQ3p1c3akg49_CJlNw7mBZuWpk4SI3fu3x1lEDpkt72-YW_tbJ5xLZQwEe5cBNjvb0ltq0b0AtGtIDDaK6w2uZbJBglPBmrYQdF2CwGu9LEO71EaVdm6kTdtguLE0-p7zxFyXvk2kpFdXtHmrwk0DOGpLVrDrnp5NVdbbewz&c=22475869&co=2798269079
